### PR TITLE
fix(alert-group): update examples and documentation

### PR DESCRIPTION
**Pull request by jessiehuff**
Closes: #854

### CODE REVIEW

File: src/patternfly/components/AlertGroup/examples/AlertGroup.md [hunk 1]
```diff
@@ -112,7 +112,13 @@ cssPrefix: pf-c-alert-group
 ### Overview
 An alert group that includes the `.pf-m-toast` modifier becomes a toast alert group with unique positioning in the top-right corner of the window. `.pf-c-alert-group` is required to create a toast alert group.
 
-Every toast alert must include a close button to dismiss the alert.
+Every toast alert must include a close button to dismiss the alert. <br><br>
```
Comment (mcoker) [inline src/patternfly/components/AlertGroup/examples/AlertGroup.md:115]: do you need these or could you just put a couple of new lines between this sentence and the next one?

File: src/patternfly/components/AlertGroup/examples/AlertGroup.md [hunk 2]
```diff
@@ -112,7 +112,13 @@ cssPrefix: pf-c-alert-group
 ### Overview
 An alert group that includes the `.pf-m-toast` modifier becomes a toast alert group with unique positioning in the top-right corner of the window. `.pf-c-alert-group` is required to create a toast alert group.
 
-Every toast alert must include a close button to dismiss the alert.
+Every toast alert must include a close button to dismiss the alert.  
+Dynamic alerts could include a link or action, as long as it's incorporated into the message in a way that users would know how to access that link/action when the dynamic alert disappears. This is to accommodate users who will not be able to access the actions in a dynamic alert. 
+
+For example:
+- "The build is complete. Go to the Builds page to `<button class="pf-c-button pf-m-link pf-m-inline>`download`</button>.`"
```
Comment (mcoker) [inline src/patternfly/components/AlertGroup/examples/AlertGroup.md:119]: We treat the opening and closing tags and anything in between as code.

```suggestion
- "The build is complete. Go to the Builds page to `<button class="pf-c-button pf-m-link pf-m-inline>download</button>.`"
```

File: src/patternfly/components/AlertGroup/examples/AlertGroup.md [hunk 3]
```diff
@@ -112,7 +112,13 @@ cssPrefix: pf-c-alert-group
...
+
+- "The build is complete. Go to the `<a href="url">`Builds`</a>` page to download."
```
Comment (mcoker) [inline src/patternfly/components/AlertGroup/examples/AlertGroup.md:121]:
```suggestion
- "The build is complete. Go to the `<a href="url">Builds</a>` page to download."
```

File: src/patternfly/components/AlertGroup/examples/AlertGroup.md [hunk 4]
```diff
@@ -98,7 +98,7 @@ cssPrefix: pf-c-alert-group
         Info toast alert title
       {{/alert-title}}
       {{#> alert-description}}
-        Info toast alert description. <a href="#">This is a link.</a>
+        Info toast alert description. From the settings tab, click <a href="#">View logs</a>&nbsp;to review the details
```
Comment (mcoker) [inline src/patternfly/components/AlertGroup/examples/AlertGroup.md:101]: Do we need a period here?

```suggestion
        Info toast alert description. From the settings tab, click <a href="#">View logs</a>&nbsp;to review the details.
```

**Review by mcoker — APPROVED** [review]
Super duper!

**Comment by jgiardino** [comment]
Thanks for making these updates, @jessiehuff!

Some quick and hopefully easy feedback...

1. I'd suggest adding a space between the 2nd and 3rd paragraph for more visual separation.
    image

2. The contents I originally drafted don't fit well with the rest of the contents. Current contents call them "toast alerts", I called them "dynamic alerts" way back. And re-reading what I originally wrote, in this context, I think it helps to explain what happens when interactive elements are included (i.e. not announced as interactive). Here's a suggestion for rewriting the 3rd paragraph to address these points:
    > When toast alerts include a link or action, these elements are not announced as interactive elements by screen readers. Whenever you include a link or button, incorporate it into the message so that it’s clear to the user how to access the same contents or action without clicking the link or button directly in the toast. For example, if your toast alert displays a message “The build is complete. Go to the Builds page to download,” the screen reader user is given instructions on how to find their build for download.

3. In my suggestion above, I ended with a "For example," that includes the example message without markup included. I find the markup embedded in the statement makes it hard to read the statement, and fully grasp the explanation. So adding it at the end of the 3rd paragraph hopefully clarifies the recommendation that users need instruction on where to go.
4. Glancing at the bullet points, it's not obvious why I would choose one over the other. Here's an alternate way of presenting this information:
    > For sighted users, interactive elements can be included in this message in one of the following ways:
    > - Using a link to the Builds page: “The build is complete. Go to the [Builds](#) page to download” using `<a href="url">Builds</a>`
    > - Using a button to download: “The build is complete. Go to the Builds page to [download](#)” using `<button class="pf-c-button pf-m-link pf-m-inline>download</button>`

**Pull request merged**